src/EntityFramework.SqlServerCompact/EntityFramework.SqlServerCompact.csproj

```diff
@@ -65,8 +65,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.SqlServer.Compact" Version="4.0.8876.1" />
-
-    <PackageReference Include="System.Data.SqlServerCe_unofficial" Version="4.0.8482.1" />    <Reference Include="Microsoft.CSharp" />
```

Comment:
Doesn't look like this reference is actually needed in this project, and the fact that we were using an unofficial package is also suspect.

Reply:
You should actually get rid of SQL Server Compact entirely - out of support since 2021 - https://learn.microsoft.com/en-us/lifecycle/products/microsoft-sql-server-compact-40

Reply:
That's a good piece of information but is outside of what this change is attempting to do.  When the time is right, we will keep this in mind.  Thanks!